src/Uno.UI.Dispatching/Native/NativeDispatcher.cs

```diff
@@ -439,7 +425,7 @@ private static EventActivity LogScheduleEvent(Action handler, NativeDispatcherPr
 		/// </summary>
 		internal void WakeUp()
 		{
-			CheckThreadAccess();
+			// CheckThreadAccess();
```

Comment:
Is this expected ?

Reply:
Yes, WakeUp can be run from a separate thread.